spython/logger/compatibility.py

```diff
@@ -0,0 +1,6 @@
+def decodeUtf8String(inputStr):
+    """Convert an UTF8 sequence into a string
+
```

Comment:
Same here for docstring (add arguments and fix quotes). Also add license header.